index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gergito Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    
+    <header>
+        <nav id="top-nav">
+          <a class="logo" href="#">My Logo</a>          
+          <nav><img class="menu" src="img/menu.png" alt="Menu" ></nav>
+        </nav>
+
+    </header>    
+    <section id="home_content">
+        <h1>I'm Gerg
+         Glad to see you!</h1>
+         <p>
+             I'm a remote software developer! I can help you build a product , 
+             feature or website Look through some of my work and experience! 
+             If you like what you see and have a project you need coded, don't hestiate to contact me.
+         </p>
+         <footer>
+             <h3>LET'S CONNECT</h3>
```

Comment:
- [ ] Kindly make sure you use h3 only if you already have h2 to meet the semantic HTML rules :recycle:

Reply:
> * [ ]  Kindly make sure you use h3 only if you already have h2 to meet the semantic HTML rules ♻️

Accepted and corrected